test/plugins/docker.test.js

```diff
@@ -1,38 +1,38 @@
 import assert from 'assert';
-import dockerImage from '../../lib/resolver/docker-image.js';
+import dockerImage from '../../lib/plugins/docker';
```

Comment:
For simplicity and keeping the PR small as possible, I just fixed the broken tests. I'll polish all plugin tests in a separate PR.